### DISCUSSION TITLE
Makefile error

### DISCUSSION THREAD

**Discussion by ADerSchreiber** [AlexeyAB/darknet#7477] [Q&A]
Hi :-)

I am currently following [this tutorial](https://youtu.be/rHY3T7-vK38) presenting the implementation of face tracking. I am currently stuck at [the makefile part](https://www.youtube.com/watch?v=rHY3T7-vK38&t=742s). Since I work on Windows 10, I got the 'make' command to work, and tried executing make in the respective directory (after inserting the values necessary for my hardware etc. in the makefile).

However, I consistently get the follow error output: 

_D:\01_Lehre\Projektphase\Tello EDU\Facetracking\Tello-AI-YOLO\Darknet>make

process_begin: CreateProcess(NULL, uname, ...) failed.
Makefile:22: pipe: No error
mkdir -p ./obj/
The syntax of the command is incorrect.
make: *** [Makefile:185: obj/] Error 1_

I am attaching the full makefile. Since this is my first time working with makefiles in this way, I tried Hail Marys by rewriting some lines with no luck so far. 

I'd greatly appreciate any advice, thank you in advance. 

[Makefile_Backup.txt](https://github.com/AlexeyAB/darknet/files/6100359/Makefile_Backup.txt)

**Comment by stephanecharette** [comment]
Windows doesn't support `make`.  Instead, you should look at the project's readme to see the various ways to build Darknet on Windows, including vcpkg, the powershell script, or using a Visual Studio solution file.

**Reply by ADerSchreiber** [reply]
Will do, thanks for the quick reply.
